Results list block main

RohitTWP opened the pull request:
Results List Block for News Theme

RohitTWP committed:
PEN-469 results-list default and simple

RohitTWP committed:
update readme and documentation

RohitTWP committed:
code review comments fixes

- Used date block for article date in results list
- Removed console.log in byline block
- Fixed the way state is set for Byline
- A prop called 'date' can be passed to the date block so thata it can be reused in other components
- added date-block as a dependency for results-list-block
- used font-families from blocks.json for header and description text for results list

RohitTWP committed:
remove console

RohitTWP committed:
credits fix

RohitTWP committed:
fix href for list items

brentmiller1973 reviewed (commented):
I think we should take another look at this.  I think if we should be doing the conditional style logic with-in the scope of styled components (See: https://stackoverflow.com/questions/48502647/conditional-rendering-in-styled-components)

RohitTWP committed:
code review fixes

- added more tests to ResultsList
- used style components in Byline to apply conditional styling
- Fixed tests in Byline

RohitTWP committed:
Merge branch 'master' into results-list-block-main

RohitTWP committed:
test names

RohitTWP committed:
fix

RohitTWP committed:
missed fix

jgrosspietsch committed:
Update blocks/results-list-block/features/results-list/default.test.jsx

jgrosspietsch reviewed (approved):
:cool:
